.pre-commit-config.yaml

```
@@ -7,8 +7,8 @@ ci:
     autoupdate_commit_msg: 'ci(deps): pre-commit autoupdate'
     autoupdate_schedule: weekly
     skip:
-      - "ruff"
-      - "poetry-lock"
+      - "ruff-check"
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)